flink-table/flink-table-planner/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -158,16 +160,17 @@
  * Default implementation of {@link SqlValidator}, the class was copied over because of
  * CALCITE-4554.
  *
- * <p>Lines 1958 ~ 1978, Flink improves error message for functions without appropriate arguments in
+ * <p>Lines 1961 ~ 1981, Flink improves error message for functions without appropriate arguments in
  * handleUnresolvedFunction at {@link SqlValidatorImpl#handleUnresolvedFunction}.
  *
- * <p>Lines 3736 ~ 3740, Flink improves Optimize the retrieval of sub-operands in SqlCall when using
- * NamedParameters at {@link SqlValidatorImpl#checkRollUp}.
+ * <p>Lines 3739 ~ 3743, 6333 ~ 6339, Flink improves validating the SqlCall that uses named
+ * parameters, rearrange the order of sub-operands, and fill in missing operands with the default
```

Comment:
can you please create a corresponding issue for Calcite to make it supported there as well?

Reply:
Thank you for the suggestion. I have created the corresponding JIRA ticket. You can find it at https://issues.apache.org/jira/browse/CALCITE-6245.